plugins/packages/restapi/lib/index.ts

```diff
@@ -119,10 +120,17 @@ export default class RestapiQueryService implements QueryService {
     const method = queryOptions['method'];
     const json = method !== 'get' ? this.body(sourceOptions, queryOptions, hasDataSource) : undefined;
     const paramsFromUrl = urrl.parse(url, true).query;
+
+    if (authType === 'bearer') {
+      headers['Authorization'] = `Bearer ${sourceOptions.bearer_token}`;
+    }
+
     try {
       const response = await got(url, {
         method,
         headers,
+        username: authType === 'basic' && sourceOptions.username,
```

Comment:
please conditionally add the keys, else this can send username/pass as false

Reply:
in GOT npm doc, to remove username and password we should pass an undefined or something, so in the above code if the username/password isn't existed it will be set as undefined